releasenotes/notes/use-cloud-provider-ntp-2995657d00d3007d.yaml

```diff
@@ -0,0 +1,4 @@
+enhancements:
+  - |
+    The NTP check now uses the cloud provider's NTP server by default, and Google's public servers
+    if no cloud provider is detected.
```

Comment:
> if no cloud provider is detected.

`IsCloudProviderEnabled` does not detect cloud provider, it reads `cloud_provider` from datadog.yaml.